app/packages/core/src/plugins/SchemaIO/components/FileView.tsx

```diff
@@ -1,43 +1,65 @@
-import { Box } from "@mui/material";
+import { Alert, Box, Typography } from "@mui/material";
 import React, { useState } from "react";
 import FileDrop from "./FileDrop";
 import HeaderView from "./HeaderView";
 import TabsView from "./TabsView";
 import TextFieldView from "./TextFieldView";
 import { getComponentProps } from "../utils";
+import { humanReadableBytes } from "@fiftyone/utilities";
 
 export default function FileView(props) {
   const { onChange, path, schema, autoFocused } = props;
   const { view = {} } = schema;
   const { types } = view;
   const [type, setType] = useState("file");
+  const isObject = schema.type === "object";
+  const maxSize = view.max_size || null;
+  const customMaxSizeMessage = view.max_size_error_message || null;
+  const [currentError, setCurrentError] = useState(null);
+
+  const showError = (message) => {
+    setCurrentError({ message });
+  };
 
   return (
     <Box {...getComponentProps(props, "container")}>
       <HeaderView {...props} nested />
-      <TabsView
-        schema={{
-          view: {
-            choices: [
-              { value: "file", label: "Upload" },
-              { value: "url", label: "URL" },
-            ],
-          },
-        }}
-        onChange={(path, value) => {
-          setType(value);
-          onChange(path, "");
-        }}
-        {...getComponentProps(props, "tabs")}
-      />
       <Box sx={{ pt: 1 }} {...getComponentProps(props, "fileContainer")}>
         {type === "file" && (
           <FileDrop
-            onChange={async (files) => {
-              if (files?.length === 0) return onChange(path, "");
+            onChange={async (files, clear) => {
+              if (files?.length === 0) {
+                return onChange(path, "");
+              }
               const [file] = files;
               const { error, result } = await fileToBase64(file);
-              if (error) return; // todo: handle error
+              if (error) {
```

Comment:
nit: when `FileView` is used with an `Object` property and the value is `None`, the component currently shows "Invalid object" error message. Ideally it would show "Required property" instead.

Reply:
This is due to the default value being an empty string. Fixed